demo/cancel_events/run.py

```diff
@@ -1,10 +1,15 @@
 import time
 import gradio as gr
+import atexit
+import pathlib
 
+log_file = (pathlib.Path(__file__).parent / "cancel_events_output_log.txt").resolve()
 
 def fake_diffusion(steps):
+    log_file.write_text("")
     for i in range(steps):
         print(f"Current step: {i}")
+        log_file.write_text(f"Current step: {i}\n")
         time.sleep(0.5)
```

Comment:
Could we make this faster? So that we could reduce the time that we have to wait in the test
```suggestion
        time.sleep(0.2)
```